tests/integration/test_install_basic.py

```diff
@@ -92,6 +92,23 @@ def test_install_without_dev(pipenv_instance_private_pypi):
 
 @pytest.mark.basic
 @pytest.mark.install
+def test_install_with_version_req_default_operator(pipenv_instance_pypi):
+    """Ensure that running `pipenv install` work when spec is package = "X.Y.Z". """
+    with pipenv_instance_pypi(chdir=True) as p:
+        with open(p.pipfile_path, "w") as f:
+            contents = """
+[packages]
+fastapi = "0.95.0"
+            """.strip()
+            f.write(contents)
+        c = p.pipenv("install")
+        assert c.returncode == 0
+        assert "fastapi" in p.pipfile["packages"]
+
+
+@pytest.mark.basic
+@pytest.mark.install
+@pytest.mark.resolver
```

Comment:
Why adding `pytest.mark.resolver` ?

Reply:
Pasting issue trying to make test running locally. Fixing